Marlin/src/gcode/host/M360.cpp

```diff
@@ -182,7 +182,7 @@ void GcodeSuite::M360() {
   config_line(F("NumExtruder"), EXTRUDERS);
   #if HAS_EXTRUDERS
     EXTRUDER_LOOP() {
-      config_line_e(e, JERK_STR, TERN(HAS_LINEAR_E_JERK, planner.max_e_jerk[E_INDEX_N(e)], TERN(HAS_CLASSIC_JERK, planner.max_jerk.e, DEFAULT_EJERK)));
```

Comment:
Can be simplified to:
config_line_e(e, JERK_STR, TERN(HAS_CLASSIC_JERK, planner.max_jerk.e, planner.max_e_jerk[E_INDEX_N(e)]));

Reply:
From what I can gather, `HAS_LINEAR_E_JERK` refers to an E jerk calculation that is derived from a combination of `planner.junction_deviation_mm` and `planner.settings.max_acceleration_mm_per_s2`, as calculated in `planner.recalculate_max_e_jerk`. Furthermore, with `HAS_LINEAR_E_JERK` there's a separate jerk value for each extruder when `DISTINCT_E_FACTORS` is enabled. These are stored in an array named `planner.max_e_jerk[]`.

Meanwhile, when `CLASSIC_JERK` is enabled, there's only one E jerk value, and it's stored in the `planner.max_jerk` object. It is set directly rather than being derived from the JD value.

These distinctions are important to the current implementation, so before making any further suggestions about refactoring E jerk handling you'll want to look at all the PRs and discussions from the period when `HAS_LINEAR_E_JERK` was first being developed in response to Junction Deviation and Linear Advance.